projects/eval-adxl313z/src/platform/stm32/main.c

```diff
@@ -75,6 +85,12 @@ int main()
 		goto error;
 #endif
 
+#if (IIO_EXAMPLE + DUMMY_EXAMPLE > 1)
```

Comment:
this won't work as expected seeing how BASIC_EXAMPLE is used instead of DUMMY_EXAMPLE and DUMMY_EXAMPLE is never defined.
replace DUMMY_EXAMPLE with BASIC_EXAMPLE.